lightning/src/chain/channelmonitor.rs

```diff
@@ -3579,7 +3579,12 @@ mod tests {
 				let sighash = hash_to_message!(&$sighash_parts.signature_hash($idx, &redeem_script, $amount, SigHashType::All)[..]);
```

Comment:
Here the sighash is computed with `All`, but below the sighash is conditional.

Reply:
good catch!  fixed ...